### PR TITLE
fix #889: implement resend email functionality

### LINKED ISSUE #889: Resend verification email doesn't seem to send the verification

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -26,13 +26,33 @@ function removeEmail(req, res) {
...
 }
 
-function resendEmail(req, res) {
-  // console.log(req.body.emailId);
+async function resendEmail(req, res) {
+  const emailId = req.body.emailId;
+  const sessionUser = _requireSessionUser(req);
+  const existingEmail = await DB.getEmailById(emailId);
+
+  if (existingEmail.subscriber_id !== sessionUser.id) {
```
[the remark this answers is not in the document]
Comment (groovecoder) [reply controllers/user.js]: Nope, not at all guaranteed. Will update to check `existingEmail` first.

File: controllers/user.js [hunk 2]
```diff
@@ -26,13 +26,33 @@ function removeEmail(req, res) {
...
+      whichView: "email_partials/email_verify",
+    }
+  );
 
-  // Resend verification email
-  // Delete the unverified email if it appears anywhere else (under the user's id);
-  // Reset the verification clock
-  return res.json("Resend the email");
+  // TODO: what should we return to the client?
+  return res.json("Resent the email");
```
Comment (pdehaan) [inline controllers/user.js]: Assuming this is user facing, should it be localized?

Comment (groovecoder) [reply controllers/user.js]: This isn't user-facing yet. I'm not sure if the status response is in the UI. Whatever is user-facing will be localized though.

**Comment by groovecoder** [comment]
Merging so I can branch my next fixes off of `monitor_v2`

**Pull request merged**